examples/server-side-rendering/src/app/ssrRouter.ts

```diff
@@ -0,0 +1,63 @@
+import * as qs from 'qs';
+
+export function ssrRouter(readUrl: () => string) {
```

Comment:
move to app.component.ts file

Reply:
consistently with e-commerce demo in InstantSearch.js, I think it makes sense to have this in its own file, since it's fairly big